### PR TITLE
Add M10iA/7L variant URDF

### PR DESCRIPTION
**Pull request by JeremyZoss**
- visual meshes simplified (convex hull) per normal procedure
- URDF validated on physical robot

### CODE REVIEW

File: fanuc_m10ia_support/launch/test_m10ia7l.launch [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0"?>
+<launch>
+  <include file="$(find fanuc_m10ia_support)/launch/load_m10ia7l.launch" />
+  <param name="use_gui" value="true" />
```
Comment (gavanderhoorn) [inline fanuc_m10ia_support/launch/test_m10ia7l.launch]: Could you please make this a private parameter of the `joint_state_publisher` node?

**Comment by gavanderhoorn** [comment]
Thanks for the PR.

Apart from the minor thing in the `test` launch file, could you add a commit updating the [package manifest](https://github.com/ros-industrial/fanuc/blob/6c953e15c13aeaec0c2ef1cb7d675689c1e382b1/fanuc_m10ia_support/package.xml#L9-L13) to mention the new variant? See the manifest of the [lrmate200ic](https://github.com/ros-industrial/fanuc/blob/6c953e15c13aeaec0c2ef1cb7d675689c1e382b1/fanuc_lrmate200ic_support/package.xml#L9-L14) package for an example.

**Comment by gavanderhoorn** [comment]
The travis failure is due to a connection timeout btw, nothing to do with this PR.

File: fanuc_m10ia_support/launch/test_m10ia7l.launch (hunk 1 above)
Comment (JeremyZoss) [inline fanuc_m10ia_support/launch/test_m10ia7l.launch]: Will do.  I was just following the structure of the existing m10ia files.  So you may want to update `test_m10ia.launch` as well.

Comment (gavanderhoorn) [inline fanuc_m10ia_support/launch/test_m10ia7l.launch]: >  I was just following the structure of the existing m10ia files.

I know, and that was fine, but the convention of making it a private parameter was introduced after the M-10iA support package was created. I haven't had an opportunity to fix everything up.

**Comment by JeremyZoss** [comment]
@gavanderhoorn : updated as requested.  Please review.

**Comment by gavanderhoorn** [comment]
:+1:

thanks for this. Always appreciated.

**Pull request merged**